app/src/main/res/layout-sw600dp/revision_card_fragment.xml

```diff
@@ -12,7 +12,7 @@
   <ScrollView
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/color_def_oppia_light_green"
+    android:background="@color/component_color_exploration_activity_background_color"
```

Comment:
This should be renamed to `component_color_revision_card_activity_background_color `

Reply:
Done